docs/source/markdown/podman-system-reset.1.md

```diff
@@ -1,7 +1,7 @@
 % podman-system-reset(1)
 
 ## NAME
-podman\-system\-reset - Reset storage back to initial state
+podman\-system\-reset - Reset storage back to initial state. This must be run after changing `containers.conf` `volume_path`
```

Comment:
Suggested wording for down below:

``suggestion
This `podman -system-reset` command be run after changing  the following values in `containers.conf` file: `static_dir`, `tmp_dir` or `volume_path`
```

Reply:
updated